modules/datastore/modules/datastore_mysql_import/README.md

```diff
@@ -7,6 +7,6 @@ To use, simply enable _datastore_mysql_import_ and clear your cache.
 
 ## Differences in behavior with the default DKAN importer.
 
-* Any "blank" rows in your data file, including carriage returns, will be imported into the datastore as empty rows. The default importer will ignore these rows.
+* Any "blank" rows in your data file, including carriage returns, will be imported into the datastore as empty rows. The default importer ignores any blank rows.  To have the Mysql Importer behave the same as default importer and ignore empty rows, enable the option on this page /admin/dkan/datastore/mysql_import
```

Comment:
Can you add a "the" before default importer in the second sentence?

Reply:
Corrected.